src/utils/requestNavigationHandlers.ts

```diff
@@ -104,15 +103,15 @@ export const handleMobileWalletProtocolRequest = async ({
 
     if (isHandshakeAction(action)) {
       logger.debug(`Processing handshake action for ${action.appId}`);
-      const chainIds = Object.values(ChainId).filter(value => typeof value === 'number') as number[];
+
       const receivedTimestamp = Date.now();
 
       const dappMetadata = await fetchClientAppMetadata();
       return new Promise((resolve, reject) => {
         const routeParams: WalletconnectApprovalSheetRouteParams = {
           receivedTimestamp,
           meta: {
-            chainIds,
+            chainIds: SUPPORTED_MAINNET_CHAIN_IDS,
```

Comment:
we might want to use `SUPPORTED_CHAIN_IDS` here as previously there wasn't a filter on testnets - iirc we wanted to be a bit more permissive in the initial connections